Dockerfile

```diff
@@ -42,7 +42,8 @@ ENV FLASK_APP=/code/reana_server/app.py
 
 EXPOSE 5000
 
-CMD set -e && ./scripts/setup &&\
+CMD set -e &&\
+    ./scripts/setup > /var/log/reana-server-init-output.log 2>&1 &&\
```

Comment:
What about starting to centralise some of the interesting cluster outputs in `/var/reana/logs/...` common place? OT1H it may be handy for debugging in case pod is relocated and local storage dies, OTOH using PVC could be slower for live deployments, but I guess former prevails over latter.